ReactSkia/components/RSkComponentText.cpp

```diff
@@ -117,8 +127,10 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
 
             numberOfLines = getNumberOfLines(metrics, props.paragraphAttributes.maximumNumberOfLines, frame.size.height);
             if (numberOfLines){
+                if (strcmp(convertEllipsizeMode(props.paragraphAttributes.ellipsizeMode),"clip")) {
```

Comment:
is it required to convert to string to check for "clip" mode ?
can we directly check for "EllipsizeMode::Clip" ?

Reply:
yes, converting to string to check "clip" mode is not required.